packages/server/src/main/java/software/uncharted/terarium/hmiserver/configuration/Config.java

```diff
@@ -10,10 +10,12 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import lombok.experimental.Accessors;
+import org.springframework.scheduling.annotation.EnableScheduling;
 import software.uncharted.terarium.hmiserver.annotations.TSModel;
 
 @Configuration
 @ConfigurationProperties(prefix = "terarium")
+@EnableScheduling
```

Comment:
what's this for?

Reply:
Apparently Spring @Schedule needs that to be enabled.  All the examples seem to place that on the Config class... so I followed suit <shrug>